tools/wini/subprocess.py

```diff
@@ -0,0 +1,44 @@
+"""Wrappers around `subprocess` for wini that print debug info."""
+
+import subprocess
+from typing import Mapping
+
+from . import print
+
+
+def check_call(
+    cmd: list[str],
+    *,
+    cwd: str | None = None,
+    env: Mapping[str, str] | None = None,
```

Comment:
same as above... i think it was added in py3.10.
i think you can use `from __future__ import annotations`

Reply:
fixed